src/controllers/krate/metadata.rs

```diff
@@ -91,15 +90,12 @@ pub async fn find_crate(
             .ok_or_else(|| crate_not_found(&path.name))?;
 
     let mut versions_publishers_and_audit_actions = if include.versions {
-        let mut versions_and_publishers: Vec<(Version, Option<User>)> =
-            Version::belonging_to(&krate)
-                .left_outer_join(users::table)
-                .select(<(Version, Option<User>)>::as_select())
-                .load(&mut conn)
-                .await?;
-
-        versions_and_publishers
-            .sort_by_cached_key(|(version, _)| Reverse(semver::Version::parse(&version.num).ok()));
+        let versions_and_publishers: Vec<(Version, Option<User>)> = Version::belonging_to(&krate)
+            .left_outer_join(users::table)
+            .select(<(Version, Option<User>)>::as_select())
+            .order_by(versions::id)
```

Comment:
@Turbo87 Hmm, should we sort this in descending order? This more closely aligns with the previous ordering.

Reply:
ah, yeah, that probably makes sense